cmd/dockerd/config.go

```diff
@@ -85,6 +85,7 @@ func installCommonConfigFlags(conf *config.Config, flags *pflag.FlagSet) error {
 	flags.Var(opts.NewNamedListOptsRef("node-generic-resources", &conf.NodeGenericResources, opts.ValidateSingleGenericResource), "node-generic-resource", "Advertise user-defined resource")
 
 	flags.IntVar(&conf.NetworkControlPlaneMTU, "network-control-plane-mtu", config.DefaultNetworkMtu, "Network Control plane MTU")
+	flags.StringVar(&conf.HostServiceDynamicPortRange, "host-service-dynamic-port-range", "", "Port allocator range")
```

Comment:
is this only affecting (swarm) services or also for `docker run -P` ? If the latter, we should pick a different name

Reply:
Also, what is the format that this flag takes? Should this take a structured approach (`min=xx,max=xx`)?

Reply:
yes, it will impact both swarm and docker run -P, will think of a better name

Reply:
The flag is --service-port-range "50000-60000"